tests/unit/testplan/testing/test_pytest.py

```diff
@@ -0,0 +1,213 @@
+"""Unit tests for PyTest runner."""
+import collections
+import json
+import os
+
+import pytest
+
+from testplan.testing import py_test as pytest_runner
+import testplan.report
+from testplan import defaults
+
+
+@pytest.fixture
+def pytest_test_inst():
+    """Return a PyTest test instance, with the example tests as its target."""
+    # This file is at tests/unit/testplan/testing/test_pytest.py, we want to run
+    # the pytest example at examples/PyTest/pytest_tests.py.
+    root_path = os.path.join(
+        os.path.dirname(__file__), *(os.pardir for _ in range(4)),
+    )
+    example_path = os.path.join(
+        root_path, "examples", "PyTest", "pytest_tests.py"
+    )
+    return pytest_runner.PyTest(
+        name="pytest example",
+        target=example_path,
+        stdout_style=defaults.STDOUT_STYLE,
+    )
+
+
+_EXPECTED_DRY_RUN_REPORT = testplan.report.TestGroupReport(
```

Comment:
I have a tiny non-blocking preference that `_EXPECTED_DRY_RUN_REPORT` live in a different file so readers' concentration isn't broken when studying this file top to bottom.

Reply:
seems reasonable, I'll move it out.